lib/jekyll_pages_api_search/search_hook.rb

```diff
@@ -12,13 +13,36 @@
 module Jekyll
   class Site
     alias_method :pages_api_after_render, :after_render
+    alias_method :orig_write, :write
+
+    def skip_index?
+      search_config = self.config['jekyll_pages_api_search']
+      search_config == nil || search_config['skip_index']
```

Comment:
Ah, it does if `jekyll_pages_api_search:` is present in `_config.yml`. Then `skip_index:` can be used to turn off indexing without having to delete the `jekyll_pages_api_search:` entry from the file.

I thought about having default parameters for `jekyll_pages_api_search` so no updates to `_config.yml` would be required, but I figured it's best to require the explicit configuration as a form of enforced documentation.